### DISCUSSION TITLE
the model does not perform well in pubg

### DISCUSSION THREAD

**Discussion by ShawnWicky** [SunOner/yolov8_aimbot#143] [Q&A]
I tried the default .engine model on the shooting range of pubg with first person perspective. But the performance is not very good, is that because the model does not trained by pubg resoruces or maybe because i didnt fine tune the config correctly?

**Comment by SunOner** [comment]
What model are you use?

**Reply by ShawnWicky** [reply]
0.2.1_nano_320.engine

**Reply by SunOner** [reply]
Yes, this is old model. Current last version is 0.3.7.
